src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreRequest.cs

```diff
@@ -184,5 +184,7 @@ public LockFile ExistingLockFile
         public bool IsRestoreOriginalAction { get; set; } = true;
 
         public bool RestoreForceEvaluate { get; set; }
+
+        public IReadOnlyList<IAssetsLogMessage> AdditionalMessages { get; set; }
```

Comment:
Why Additional in the name? There are not any messages yet.

Reply:
Very true. I was thinking they're additional to whatever messages restore generates.